src/Form/AkamaiClearUrlForm.php

```diff
@@ -23,10 +23,9 @@ public function getFormId() {
    * {@inheritdoc}
    */
   public function buildForm(array $form, FormStateInterface $form_state) {
-    $config = $this->config('akamai.settings');
     $form = array();
 
-    $current_uri = $this->requestStack->getCurrentRequest()->getRequestUri();
+    $current_uri = $this->getRequest()->getRequestUri();
```

Comment:
I forget to inject the requestStack dependency in constructur on previous PR. The FormBase class is not injected the request object during instantiating of object.
I found another helper function getRequest function in FormBase which take care of creating an request instance for us. 
So I found its useful